spec/features/10_update_reviews_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Edit Movie" do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:movie1) { FactoryGirl.create(:movie) }
+  let!(:movie2) { FactoryGirl.create(:movie) }
+  let!(:review5) { FactoryGirl.create(:review, movie_id: movie1.id, user_id: user.id) }
+
+  before do
+    user_sign_in(user)
+  end
+
+  scenario "User clicks link edit review, and is redirected to"\
```

Comment:
This is not a single quote issue, this is a "line too long" correction to quiet Hound.